src/main/java/game/Item.java

```diff
@@ -49,10 +55,11 @@ public boolean onCommand(Command command, Callback callback) {
     }
 
     @Override
-    public List<String> listCommands(Game game, List<String> addToThisList) {
-        addToThisList.add("inspect " + name);
-        actions.listCommands(game).forEach(s -> addToThisList.add(s + " " + name));
-        return addToThisList;
+    public List<String> listHandledCommands(Game game) {
+        List<String> result = new ArrayList<>();
+        result.add("inspect " + name);
+        actions.listHandledCommands(game).forEach(s -> result.add(s + " " + name));
+        return result;
```

Comment:
Changing this and reducing the interactable interface made it better. Looks cleaner!